api/main.py

```diff
@@ -1,74 +1,72 @@
-from fastapi import FastAPI
-from pydantic import BaseModel, validator
+from typing import List
+from fastapi import Depends, FastAPI, HTTPException
+from sqlalchemy.orm import Session
 
-app = FastAPI()
+from . import crud, models, schemas
+from .database import SessionLocal, engine
+
+models.Base.metadata.create_all(bind=engine)
 
-students = [
-    {"student_id": 1, "name": "Jens"},
-    {"student_id": 2, "name": "Bob"},
-]
 
+app = FastAPI()
 
-class Student(BaseModel):
-    student_id: int
-    name: str
 
-    @validator("student_id")
-    def student_id_positive(cls, value):
-        if value <= 0:
-            raise ValueError(f"Expected positive price, received {value}")
-        return value
+# Dependency
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close
 
-    @validator("student_id")
-    def student_id_unique(cls, value):
-        if value in [student["student_id"] for student in students]:
-            raise ValueError(f"ID is already in use")
-        return value
 
 @app.get("/")
 def read_root():
     return {"Hello": "World"}
 
 
-@app.get("/items/{item_id}")
-def read_item(item_id: int, q: str):
-    name = q.upper()
-    return {"item_id": item_id, "q": name}
-
-
-@app.get("/name/{q}")
-def read_name(q: str):
-    return q.upper()
-
-
-@app.get("/students")
-def read_students():
-    return students
-
-
-@app.post("/students")
-def create_student(student: Student):
-    return student
-
-
-@app.put("/students/{student_id}")
-def save_student(student: Student):
-    return student
-
-
-@app.patch("/students")
-def edit_student(student: Student):
-    for person in students:
-        if person["student_id"] == student.student_id:
-            person["name"] = student.name
+@app.post("/users/", response_model=schemas.User)

```

Comment:
super minor comment :smiley: : for the sake of consistency either keep or delete "/" in all endpoints